platzky_redirections/plugin.py

```
@@ -53,11 +28,10 @@ def parse_redirections(config: dict[str, str]) -> list[Redirection]:
     """
 
     def validate_url(url: str) -> bool:
+        """Check that a URL starts with '/' or 'http'."""
         return url.startswith("/") or url.startswith("http")
```

Ruff (0.15.0)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)